PinetimeFlasher.pyw

```diff
@@ -235,9 +250,82 @@ class ConfDialog(QDialog):
         except OSError:
             self.status.setText("Unable to write configuration file!")
 
-    def infoButton(self, s):
-        dlg = InfoDialog()
-        dlg.exec()
+    def setup_openocd(self):
+        """
+        Download and unpack OpenOCD for the current platform, then add it to the system path.
+        """
+        self.status.setText("Finding latest OpenOCD release...")
+
+        base_url = "https://api.github.com/repos"
+        response = requests.get(f"{base_url}/xpack-dev-tools/openocd-xpack/releases/latest")
+
+        details = response.content.decode('utf-8')
+        content = json.loads(details)
+
+        archive_file, hash_file = self.get_github_assets(content["assets"])
```

Comment:
I've not looked any further yet than running this, but whilst the code worked perfectly on Linux (x64),
it is breaking on Windows :(
```
Traceback (most recent call last):
  File "e:/repos/GitHub/pfeerick-PinetimeFlasher/PinetimeFlasher.pyw", line 265, in setup_openocd
    archive_file, hash_file = self.get_github_assets(content["assets"])
TypeError: cannot unpack non-iterable NoneType object
```

Reply:
Just in case this is Python version specific, I'm on 3.8.6 on Windows atm... If it's not an issue with 3.9 I have no problem updating.